aws/api-servers.tf

```diff
@@ -31,11 +31,10 @@ resource "aws_elb" "api-ext" {
     lb_protocol = "http"
   }
   listener {
-    instance_port = 8080
-    instance_protocol = "http"
+    instance_port = 443
+    instance_protocol = "tcp"
```

Comment:
I propose that we shouldn't since:

*  It feels a little redundant to implement a temporary workaround for GCE which we will fix in time 
* We are adding unnecessary configuration in AWS so it matches a temporary deficiency in GCE. 

Though if you feel strongly about it we can add it in.